modules/ephemeral-storage-using-a-sharedconfimap-object-in-a-pod.adoc

```diff
@@ -0,0 +1,82 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="ephemeral-storage-using-a-sharedconfigmap-object-in-a-pod_{context}"]
+= Using a SharedConfigMap instance in a pod
+
+[role="_abstract"]
+To access a `SharedConfigMap` custom resource (CR) instance from a pod, you grant a given service account RBAC permissions to use that `SharedConfigMap` CR instance.
+
+.Prerequisites
+
+* You have created a `SharedConfigMap` CR instance for the config map that you want to share across namespaces in the cluster.
+* You must have permission to perform the following actions:
+** Discover which `SharedConfigMap` CR instances are available by entering the `oc get sharedconfigmaps` command and getting a non-empty list back.
+** Determine if the service account your pod specifies is allowed to use the given `SharedSecret` CR instance. That is, you can run `oc adm policy who-can use <identifier of specific SharedSecret>` to see if the service account in your namespace is listed.
```

Comment:
```suggestion
** Determine if the service account your pod specifies is allowed to use the given `SharedSecret` CR instance. That is, you can run the `oc adm policy who-can use <identifier_of_specific_sharedsecret>` command to see if the service account in your namespace is listed.
```
In this one, you've got "That is", while in the [other bullet elsewhere like this](https://github.com/openshift/openshift-docs/pull/82993/files#diff-4b62d7143739c02d1caa4cf0c6fc8ea1bc5521e79f6350a770914d7c04816e7aR20), it is "In other words". Suggest syncing up on one or the other.